src/dune_rules/coq_rules.ml

```diff
@@ -178,9 +178,19 @@ end
 (* get_libraries from Coq's ML dependencies *)
 let libs_of_coq_deps ~lib_db = Resolve.Memo.List.map ~f:(Lib.DB.resolve lib_db)
 
-let select_native_mode ~sctx ~(buildable : Buildable.t) : Coq_mode.t =
-  let profile = (Super_context.context sctx).profile in
-  if Profile.is_dev profile then VoOnly else snd buildable.mode
+let select_native_mode ~sctx ~dir (buildable : Coq_stanza.Buildable.t) =
+  match snd buildable.mode with
+  | Some x -> Memo.return x
```

Comment:
I think we still want to avoid building native objects in dev mode, it is very expensive and it doesn't make a lot of sense for users.

Reply:
At the very least we should preserve the previous behavior.

Tho, thinkin about it, native is more a property of the whole workspace / profile setup than of particular stanzas maybe, mmm, not sure.